ee/ucp/interlock/usage/interlock-vip-mode.md

```diff
@@ -4,6 +4,7 @@ description: Learn about task and VIP backend routing modes for Layer 7 routing
 keywords: routing, proxy, interlock
 redirect_from:
   - https://interlock-dev-docs.netlify.com/usage/default_backend/
```

Comment:
@ollypom redirecting from an external site actually doesn't work. I know you didn't add this - but I'll also remove. Thanks for the fix!

Reply:
External site redirect PR reverted via #8670.